pkgs/development/python-modules/openstackdocstheme/default.nix

```diff
@@ -11,21 +11,19 @@
 
 buildPythonPackage rec {
   pname = "openstackdocstheme";
-  version = "3.3.0";
+  version = "3.4.0";
   pyproject = true;
 
   # breaks on import due to distutils import through pbr.packaging
   disabled = pythonAtLeast "3.12";
```

Comment:
Note for reviewers: I will remove this disabled check in another PR for Python 3.12 support, because it requires to add distutils dependency to pbr (needs to go to staging branch).